package.json

```diff
@@ -53,5 +53,8 @@
     "concurrently": "^7.6.0",
     "prettier": "3.0.2",
     "prettier-plugin-organize-imports": "^3.2.3"
+  },
+  "dependencies": {
+    "@emotion/css": "^11.11.2"
```

Comment:
Do we really need this? If yes, I think it is better to add it in the carbon module package.json file.

Reply:
We added it while updating the option switch components. I moved the dependency to the carbon sub module in [09c65dd](https://github.com/KlimaDAO/klimadao/pull/1644/commits/09c65ddd27071a7c065ec87a976b86774b427a58)